hazelcast-client/src/test/java/com/hazelcast/client/proxy/ProxyManagerTest.java

```diff
@@ -123,4 +126,19 @@ public void testNextAddressToSendCreateRequestOnMultipleLiteMembers() {
 
         return instances;
     }
+
+    @Test(expected = OperationTimeoutException.class)
```

Comment:
Depending on client state it could differ. When cluster disconnected, if clients shutdown you need to get HazeclastClientNotActiveException. In this test, client invocation times out before client decided to shutdown. Therefore client invocation should time out with `OperationTimeoutException`